### DISCUSSION TITLE
Cant Access GitHub Account Due to 2FA

### DISCUSSION THREAD

**Discussion by peddy100** [community/community#119852] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, I'm unable to access my github account due to 2FA. I recently got a new phone cannot recover my authenticator credentials. I attempted to use the virtual assistant and the steps outlined in Recovering an Account with 2FA, but neither of those worked. Please help

**Comment by ebndev** [comment]
Hi @peddy100,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
